test/parallel/test-abortcontroller.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+require('../common');
+
+const { ok, CallTracker, throws } = require('assert');
+
+const tracker = new CallTracker();
+process.on('exit', () => tracker.verify());
+
+{
+  const ac = new AbortController();
+  ok(ac.signal);
+  ac.signal.onabort = tracker.calls();
+  ac.signal.once('abort', tracker.calls());
```

Comment:
Should it check for a specific number of calls?

Reply:
By default it watches for exactly one call